installer/bundle/bundle_skel_Linux.sh

```diff
@@ -620,6 +621,10 @@ case "$installMode" in
             if [ $? -eq 0 ]; then
                 shouldInstall_omi
                 pkg_upd_list $OMI_PKG omi $?
+            else
+                pkg_add_list $OMI_PKG omi
+                pkg_add ${ADD_PKG_QUEUE}
+                OMI_EXIT_STATUS=$?
```

Comment:
I don't think this is quite right. You're mixing ADD and UPDATE operations, and thus forcing the components to be updated independently. That's the issue we're trying to solve with this change.

Please look at [OMS version](https://github.com/Microsoft/OMS-Agent-for-Linux/blob/master/installer/bundle/bundle_skel.sh), line 772, to see how OMS handles it (which passed all testing).

Update will still install a kit if it's not currently installed in our Linux package managers, so during an update operation, you should bundle the OMI operation along with the SCX installation.

Let me know if you have further questions.